### ISSUE TITLE
Rectangles in place of "inline code" text

### ISSUE THREAD

**Issue by Splitwirez** [whistyun/Markdown.Avalonia#54]
# The problem
![image](https://user-images.githubusercontent.com/34009058/123472166-cf1ca780-d5c4-11eb-87f1-9d825298c847.png)
![image](https://user-images.githubusercontent.com/34009058/123472527-48b49580-d5c5-11eb-87bc-49ca61af79bc.png)
Inline code text appears as what I presume are "missing character" rectangles.

# Context
#### Operating system
Kubuntu Linux 20.04 x64
#### Affected projects
`Markdown.AvaloniaDemo` and `Markdown.AvaloniaFluentDemo`

# A hypothesis
I'm not entirely certain as to *why* this happens, but I did notice that, of the fonts listed in `FontFamilyCollecter.cs` (`menlo`, `monaco`, `consolas`, `droid sans mono`, `inconsolata`, `courier new`, `monospace`,`droid sans fallback`), none are installed on my setup except the last one (`droid sans fallback`)...and uh, something seems a bit...*off* about that one...

### Droid Sans Fallback weirdness
While I was checking which fonts were installed, I noticed that the font manager's preview of `droid sans fallback` looked rather peculiar, as shown here:
![image](https://user-images.githubusercontent.com/34009058/123473143-28d1a180-d5c6-11eb-9cc9-5cf6d22469b8.png)

For comparison, here's a more typical font preview, for comparison (specifically of `FreeSans`):
![image](https://user-images.githubusercontent.com/34009058/123473240-4b63ba80-d5c6-11eb-8e4d-643ccc81f7af.png)

### Testing in LibreOffice Writer
Oddly enough though, `Droid Sans Fallback` is perfectly readable in LibreOffice Writer, as shown here:
![image](https://user-images.githubusercontent.com/34009058/123473916-3f2c2d00-d5c7-11eb-9b80-a9402ef9ab3c.png)
(...though, given what comes next, I'm not fully convinced that LibreOffice Writer is actually rendering `Droid Sans Fallback` at all here...)

### Testing in Microsoft Office Word 2010
...after seeing this bizarre discrepancy, I decided to try comparing the fonts in some other programs. First pick was Microsoft Office Word 2010, which produces some rather...telling results:
![image](https://user-images.githubusercontent.com/34009058/123473867-2de32080-d5c7-11eb-97ff-46147957cdac.png)

# A potential fix
Just before submitting this issue, an idea popped into my head. I decided to try adding a font I knew I had installed - and which I knew would not misbehave the way `Droid Sans Fallback` did - to the list of fonts in `FontFamilyCollecter`, to see if that would help. First, I tried `FreeMono` (which I spelled as `freemono` in the code)...but this produced the same results as `Droid Sans Fallback`, no matter where I put it in the list. After reverting this, I instead tried adding `DejaVu Sans Mono` (which I spelled as `dejavu sans mono` in the code) to the list, placing it at index 0. I was pleasantly surprised when I built and ran the sample app with the changes, and was met by this:
![image](https://user-images.githubusercontent.com/34009058/123475330-3f2d2c80-d5c9-11eb-828a-74b811573ba7.png)
Further experimentation showed that I could make `dejavu sans mono` the second-last entry in the list, and the text would still be displayed correctly as shown just above. To avoid any possibility of ambiguity, that final test (which was successful) had the font list modified as follows:
```cs
            string[] RequestFamilies = {
                "menlo",
                "monaco",
                "consolas",
                "droid sans mono",
                "inconsolata",
                "courier new",
                "monospace",
                "dejavu sans mono",
                "droid sans fallback"
            };
```

# Closing note
I hope this gives you whatever you need to look into this, if/whenever you're able and willing. If you need anything more from me, feel free to @Splitwirez and I'll see what I can do next time I have a spare moment. Pretty cool library otherwise.

Oh and uh...I've not seen it spelled "Collect**E**r" before, only "Collect**O**r"...is that a regional thing?

**Comment by Splitwirez** [comment]
Just tried https://github.com/whistyun/Markdown.Avalonia/tree/dd13a2d3459fa0e5ec7986309f9ef1b4be274122
![image](https://user-images.githubusercontent.com/34009058/123558432-358bfc00-d764-11eb-8526-0d524609926f.png)
Works without a hitch now :ok_hand:

**Comment by whistyun** [comment]
I released v0.10.5 for fix it.

**Comment by dayAndnight2018** [comment]
You alse could rewrite the style of the mardownscrollviewer control to custom the font family of your inline code

**Comment by IhateTrains** [comment]
> Thanks for reporting the issue and checking my commit. I released v0.10.5 for fix it.

Shouldn't the issue be closed?
